Documentation/articles/content/why_content_pipeline.md

```diff
@@ -0,0 +1,59 @@
+
+> Note that 262 KB  is quite a bit bigger than the compressed size.
+
+The next thing to do is create a texture for that data because your device cannot compress on the fly (yet), it has to use that data as is. So in creating the texture we used 262kb  of graphics memory on the GPU. That doesn’t sound too bad, but if you are using larger textures say 1024×1024 then you are using 4 MB of GPU memory for that one texture. Multiply that over the number of textures in your game and you soon run out of texture memory on the GPU. Then the GPU has to swap that data out into system memory (if it supports that) or throw an error when you try to create textures that won’t fit into available memory. 
+
+> So to sum up:
+>
+> using **.pngs** = smaller package size & higher memory usage & less textures
+
+Now let us look at a texture that has been pre-processed using the content pipeline, because we know we are targeting iOS we know the GPU’s on those devices support using PVRTC texture compression directly. So we take our sample .png and compress it at build time using PVRTC and we end up with is a 32kb file (size depends on the texture, alpha channel, etc). Hmm, that is a lot bigger than the .png on disk, but that is not the whole story. The difference here is that there is no need to unpack/decompress it which saves on load time, also, we can create a texture from that data directly so we only use 32kb of texture memory on the GPU and not 262kb. That is a massive saving.
+
+> Summing up:
+>
+> **compressed textures** = larger package size (maybe) & lower memory usage & more textures
+
+Now we only looked at iOS, but the same applies to desktop environments, most desktop GPUs support DXT texture compression, so the content pipeline will produce DXT compressed textures which can be loaded and used directly. The only platform that is a pain is Android because the Android platform does not have consistent support for compressed textures at the moment and MonoGame has to decompress DXT textures on the device and use it directly. However, even Android will get compressed texture support eventually. There is also a piece of work happening where the pipeline tool will automatically pick a texture format to use, so for opaque textures, it will use ETC1 (which is supported on all android devices but doesn’t support alpha channels) but for textures with an alpha channel, it will use RGBA4444 (dithered) but also allow the user to pick from a wide variety of compression options manually such as PVRTC, ATITC, DXT/S3TC, ETC1, and RGBA4444. This will give the developer the choice of what to use/support.
```

Comment:
> There is also a piece of work happening where the pipeline tool will automatically pick a texture format to use,

I am pretty certain this already works.  Use the `Compressed` setting which i think is the default now?